test/e2e/parallel/proxy_test.go

```diff
@@ -1064,22 +1071,22 @@ func TestSpaceLister(t *testing.T) {
 			err := proxyClDefault.Get(context.TODO(), workspaceNamespacedName, &toolchainv1alpha1.Workspace{})
 
 			require.Error(t, err)
-			require.True(t, k8serr.IsForbidden(err), "expected Forbidden error, got %v", err)
+			require.ErrorContains(t, err, "user access is forbidden")
```

Comment:
I've tried to explain the change from error equals to errorContains in the PR description.
It's mostly because of the rest mapper change that the error is now returned wrapped from server.

Please let me know if something is unclear and can be improved.